src/components/input/input.tsx

```diff
@@ -41,7 +41,7 @@ export const Input = ({ label, error, className, type, ...rest }: Props) => {
           </button>
         )}
       </div>
-      {error && <span className={styles.errorText}>{error}</span>}
+      {error && <span className={error ? styles.error : ''}>{error}</span>}
```

Comment:
```suggestion
      {error && <span className={styles.error}>{error}</span>}
```

по твоему условию, если !error, то этот спан вообще не отрисуется. так что мы не можем попасть в правую часть этого тернарника